Assets/README.md

```diff
@@ -0,0 +1,6 @@
+# Assets
+
+This directory contains images and other assets to support the dashboards shared in this repository. Each Dashboard should have at least 1 thumbnail provided with data to showcase the underlying analytics.
```

Comment:
For `Assets`, should we create a separate folder for `Thumbnails`? If each report will have at least one thumbnail, it might make the structure more organized in the future.

Also, we should spell out `1` as `one`.

Reply:
I think I agree. working on it